src/main/java/org/helpberkeley/memberdata/DriverPostFormat.java

```diff
@@ -118,6 +119,17 @@ protected abstract ProcessingReturnValue processLoopListRef(
     protected abstract String versionSpecificSimpleRef(MessageBlockContext context, String varName);
     protected abstract String versionSpecificDriverListRef(MessageBlockContext context, String varName);
 
+    public final void generateOpsManagerPhone() {
+        ControlBlock controlBlock = getControlBlock();
+        ControlBlock.OpsManager opsManager = controlBlock.getFirstOpsManager();
+        if (opsManager.getPhone().isEmpty()) {
+            User user = users.get(opsManager.getUserName());
+            if (user != null) {
+                opsManager.setPhone(user.getPhoneNumber());
+            }
```

Comment:
I don't like that it can be null here.  We still haven't audited the ops manager user name.  Feels like this is not the best place to do this.  Looking again makes me think that it would be better to do the optional phone number support in ControlBlock.auditOpsManager() where we are being passed a User map.